src/main/java/seedu/duke/profile/Profile.java

```diff
@@ -0,0 +1,48 @@
+package seedu.duke.profile;
+
+public class Profile {
+    private int height;
+    private int weight;
+    private int calorieGoal;
+
+    public Profile(int height, int weight) {
+        setHeight(height);
+        setWeight(weight);
+        setCalorieGoal(0); //Initialize to 0 first
+    }
+
+    public Profile() {
+        setHeight(0);
+        setWeight(0);
+        setCalorieGoal(0);
+    }
+
+    public void setHeight(int height) {
+        this.height = height;
+    }
+
+    public void setWeight(int weight) {
+        this.weight = weight;
+    }
+
+    public void setCalorieGoal(int calorieGoal) {
+        this.calorieGoal = calorieGoal;
+    }
+
+    public double calculateBmi() {
+        double heightInM = ((double) height) / 100.0;
+        double bmi = Math.round((weight / (Math.pow(heightInM, 2))) * 10) / 10.0;
+        return bmi;
+    }
+
+    public static double calculateBmi(int height, int weight) {
```

Comment:
I think that we could use a `double` type for `height` and `weight` instead to allow for more precise numbers!